.github/workflows/ci.yml

```diff
@@ -24,6 +24,8 @@ env:
   MINIMAL_RUST: '1.71.0'  # Minimal Supported Rust Version
   HWLOC_VERSION: '2.10.0'
   HWLOC_VERSION_SHORT: '2.10'  # Used in URL to official tarball/binaries
+  HWLOC_DEBUG_VERBOSE: 0  # Set to 1 when debugging issues inside of hwloc
+  HWLOC_HIDE_ERRORS: 0  # Remove if this proves too chatty
```

Comment:
Better tradeoff between having the ability to see hwloc debug messages and not spamming CI when unneeded.